src/sink/elastic.rs

```diff
@@ -202,6 +209,7 @@ fn build_bulk_success_data(
     let mut meta = Object::with_capacity(1);
     let mut es_meta = Object::with_capacity(4);
     es_meta.insert("id".into(), Value::from(item.id().to_string()));
+    es_meta.insert("id".into(), Value::from(item.id().to_string()));
```

Comment:
I think here you also need to add `_id` and all.

Reply:
darn I overlooked that